src/components/KRadio/KRadio.vue

```diff
@@ -1,54 +1,79 @@
 <template>
   <div
-    :checked="isSelected"
     class="k-radio"
     :class="[
-      isTypeDefault ? 'k-radio-default' : `k-radio-${type}`,
       $attrs.class ? $attrs.class : '',
-      { 'disabled': isDisabled }
+      { 'disabled': isDisabled, 'radio-card': isCard || type === 'card', 'has-error': hasError, 'checked': isChecked, 'has-description': showDescription }
```

Comment:
suggestion: The class logic is a little complex; can we move this to a `computed` variable?